tests/test_io.py

```diff
@@ -254,14 +254,18 @@ def test_write_wmv():
 
 @no_warnings_allowed
 def test_write_quality():
+    try:
+        import numpy as np
+    except ImportError:
+        return skip("Missing 'numpy' test dependency")
 
     sizes = []
     for quality in [2, 5, 9]:
         # Prepare for writing
         gen = imageio_ffmpeg.write_frames(test_file2, (64, 64), quality=quality)
         gen.send(None)  # seed
         for i in range(9):
-            data = bytes([min(255, 100 + i * 10)] * 64 * 64 * 3)
+            data = np.random.randint(0, 255, (64, 64, 3), dtype="uint8")
```

Comment:
Tests were changed to use random because otherwise the "lossless" encoder's file size was actually smaller than the "lossy" encoders.